src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
>If there is a rational explanation (like in the Google code guide for example), I will be thankful for that. 

Is rules, supported by me a lot different? It differs only in the location of one of the header. I just don't think it's so important to have it in the beginning. This is true only for the headers of the library that contains the import. Most importantly, the entire source code of the project obeyed the same rules. Then this code becomes easy to read, easily inspected and easily maintained.
I have said that you should not impose anything stealthily
. We are open for discussions.